spring-vault-core/src/test/java/org/springframework/vault/core/VaultSysTemplateIntegrationTests.java

```diff
@@ -98,6 +99,78 @@ public void mountShouldMountGenericSecret() {
 		assertThat(Arrays.asList("kv", "generic")).contains(secret.getType());
 	}
 
+	@Test
+	public void mountShouldMountKv1Secret() {
+
+		assumeTrue(vaultVersion.isGreaterThanOrEqualTo(
+			VaultRule.VERSIONING_INTRODUCED_WITH));
+
+		if (adminOperations.getMounts().containsKey("other/")) {
```

Comment:
Let's use a more meaningful name as state persists beyond test runs and we should get an indication where this mount comes from.

Reply:
Sure, will do.